src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,93 @@
+package duke;
+
+import duke.tasks.Deadline;
+import duke.tasks.Events;
+import duke.tasks.Task;
+import duke.tasks.ToDos;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.FileWriter;
+import java.util.Scanner;
+
+public class Storage {
+
+    public static String path = "DukeData/tasks.txt";
+    public static String dirPath = "DukeData";
```

Comment:
Is this a constant? Perhaps it could be named in uppercase separated by underscores e.g. `DIR_PATH`.